README.md

```diff
@@ -1,235 +1,56 @@
-available in this repo as a git submodule. It is pulled by default during the
-`cmake` configuration step above.
-
-If a custom version of mcnp2cad is needed, this behavior pulling can be disabled
-by adding `-DUPDATE_SUBMODULES=OFF` to the `cmake` configuration. mcnp2cad can
-then be manually updated with the following commands:
-
-```
-cd ${HOME}/plugin-build/Cubit-plugin
-git submodule update --init
-```
-
-Create the Tarball
-==================
-
-The following commands show how to create the tarall for the plugin. Once again,
-the location of HDF5 might be different than what is presented here depending on
-what flavor or version of Linux is being used.
 
-```
-# Set up the directory which will contain the libraries
-cd ${HOME}/plugin-build
-mkdir -p pack/bin/plugins/svalinn
-cd pack/bin/plugins/svalinn
-
-# Copy all needed libraries into current directory
-cp -pPv /usr/lib/x86_64-linux-gnu/libhdf5_serial.so.100* .
-cp -pPv ${HOME}/plugin-build/moab/lib/libMOAB.so* .
-cp -pPv ${HOME}/plugin-build/DAGMC/lib/libdagmc.so* .
-cp -pPv ${HOME}/plugin-build/DAGMC/lib/libmakeWatertight.so* .
-cp -pPv ${HOME}/plugin-build/DAGMC/lib/libpyne_dagmc.so* .
-cp -pPv ${HOME}/plugin-build/DAGMC/lib/libuwuw.so* .
-cp -pPv ${HOME}/plugin-build/lib/* .
-chmod 644 *
-
-# The resulting shared library objects require an update to their RPATH (runtime path) 
-# attribute to ensure that the correct set of libraries is discovered when starting 
-# Cubit and loading the plugin library.
-patchelf --set-rpath PATH_TO_CUBIT/bin/plugins/svalinn libMOAB.so
-patchelf --set-rpath PATH_TO_CUBIT/bin/plugins/svalinn libdagmc.so
-patchelf --set-rpath PATH_TO_CUBIT/bin/plugins/svalinn libmakeWatertight.so
-patchelf --set-rpath PATH_TO_CUBIT/bin/plugins/svalinn libpyne_dagmc.so
-patchelf --set-rpath PATH_TO_CUBIT/bin/plugins/svalinn libuwuw.so
-
-# Create the Svalinn plugin tarball
-cd ..
-ln -sv svalinn/libsvalinn_plugin.so .
-cd ../..
-tar --sort=name -czvf svalinn-plugin-17.1.tgz bin
-mv -v svalinn-plugin-17.1.tgz ..
-cd ..
-rm -rf pack
-```
+Install Plugin
+==============
 
-The Svalinn plugin tarball should now be located at
-`${HOME}/plugin-build/svalinn-plugin-17.1.tgz`.
+The Plugin can be downloaded from the [Release](https://github.com/svalinn/Cubit-plugin/releases)
+section of this repository. Operating system specific assets are created with
+each release and can be download for various Operating systems (Ubuntu 18, 20
+and MacOS 10.15).
 
-Install the Plugin
-==================
+After downloading the compressed tgz file the contents should be uncompress to
+your Cubit folder.
+<details>
+    <summary><b>Expand</b> - Ubuntu terminal commands for Ubuntu 20.04</summary>
+    <pre><code class="language-html">
+    wget https://github.com/svalinn/Cubit-plugin/releases/download/0.1.0/svalinn-plugin_ubuntu-20.04_cubit_2021.5.tgz
+    sudo tar -xzvf svalinn-plugin_ubuntu-20.04_cubit_2021.5.tgz -C /opt/Coreform-Cubit-2021.5
```

Comment:
This is only valid for Ubuntu, I think.

Reply:
Yep I think so also, it mentions Ubuntu usage in the Expansion tab.

I did this via an expand tabs as I think we could have expand tabs for every OS once they are all working.